BowlingBall.Tests/GameFixture.cs

```diff
@@ -5,10 +5,109 @@ namespace BowlingBall.Tests
 {
     public class GameFixture
     {
+        RollingBall balls = new RollingBall();
+
+        [Fact]
+        
+        public void BasicTestForZeroScoring()
+        {
+            // Arrange
+            int actual;
+            int expected;
+
+            // Act
+            balls.RollMultipleTimes(20, 0);
+            actual = balls.GetScore();

```

Comment:
having GetScore() method on balls does not seem correct. It should be on game.
For methods like RollMultipleTimes, instead of creating a new class, you can use extension methods for the game class so that main logic is not disturbed. Do read about extension methods https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/classes-and-structs/extension-methods https://www.geeksforgeeks.org/extension-method-in-c-sharp/